src/docs/tutorials/IntroductionToMarkdown.md

```diff
@@ -191,8 +204,10 @@ These support you in
 
 [^1]: Integrated development environments
 [^2]: I am a footnote
-[^3]: <https://pandoc.org/>
+[^3]: <https://pandoc.org/> "Pandoc"
 
+----
```

Comment:
Why a double horizontal dividing line? It seems to be a break in style from the rest of the document.

Reply:
This was just concatenating two things that only half fit together.